### PR TITLE
added support for 'xmlformat' without '.pl'

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

**Comment by tomschr** [comment]
That's a cool idea!

Actually, there are two implementation of this tool: in Perl (that's the `.pl` extension) and Ruby (`.rb`). I was always unsure which one to use. That's why we have both in openSUSE.

Actually I would consider it as a `xmlformat` packaging bug. So we should  introduce a link (maybe with the `update-alternatives` tool.

File: bin/daps-xmlformat [hunk 1]
```diff
@@ -25,9 +25,16 @@ fi
 #echo $cmd
 #eval $cmd
 
-res=$(which xmlformat.pl 2>/dev/null)
+# lets decide on the binary filename
+dist=`lsb_release -ds | awk '{print $1}'`
```
Comment (tomschr) [inline bin/daps-xmlformat:29]: I'm not sure if this is the "right" test to check for Debian. Maybe it's enough to check for some Debian specific files? In openSUSE we have `/etc/SuSE-release` and `/etc/SuSE-brand`, maybe there is something similar?

Comment (tbazant) [inline bin/daps-xmlformat:29]: openSUSE has the ;lsb_release' as well, i checked it before i included it

Comment (tomschr) [inline bin/daps-xmlformat:29]: Yes, that's true. Ok, let's merge it. :-)

**Comment by tbazant** [comment]
btw Debian has only 'xmlformat' without a suffix, and it is a perl version. i consider it better to use user commands without .pl or .rb, so your suggested symlink seems a good solution

**Pull request merged**